example-projects/text/textBasic/index.js

```diff
@@ -0,0 +1,78 @@
+const THREE = require('three')
+const fontJson = require('../../fonts/droid/droid_sans_regular.typeface.json')
+
+class TextBasic {
```

Comment:
this sketch and config will need some work before getting added, but getting this up for you now so you can make a string do something right away

Reply:
Yeah makes sense. Perhaps we could provide a few fonts in the global `HEDRON` object. Otherwise I'd advise to have the font located inside the `text` folder. Also, worth noting your folder structure is a bit wrong for this example project, see the other ones to see what I mean. Although I was recently thinking we could change the structure of all this anyway: https://github.com/nudibranchrecords/hedron/issues/341

Reply:
It's probably worth doing an upstream merge to this branch so that you're up to date. You shouldn't be pulling in `THREE` like that with the new hedron, it's now a global object. `window.HEDRON.dependencies.THREE`